### ISSUE TITLE
Getting docker to work?

### ISSUE THREAD

**Issue by Trentan** [yuzawa-san/googolplex-theater#517]
Hi there - all but working (docker install) getting error:

Trying this step from cmd prompt:
docker run --net=host -v /path/to/your/conf:/opt/googolplex-theater/conf ghcr.io/yuzawa-san/googolplex-theater:VERSION

rebranded for my machine- (eg trying to point the C:Temp drive)

C:\Temp>docker run --net=host -v C:/temp yuzawa/googolplex-theater:latest

Returns error:
ParseException: cast-config file does not exist: /opt/googolplex-theater/src/dist/conf/cast_config.json
usage: googolplex-theater [-a <ABCDEFGH>] [-c <FILE>] [-h] [-i
       <IFACE_NAME_OR_IP>] [-p <PORT_NUM>]
 -a,--app-id <ABCDEFGH>              cast application ID
 -c,--cast-config <FILE>             cast config json file
 -h,--help                           show usage
 -i,--interface <IFACE_NAME_OR_IP>   network interface name or IP address
                                     to use for service discovery
 -p,--port <PORT_NUM>                server port number

I can see the missing file created in the docker volumes -
<img width="202" alt="image" src="https://github.com/yuzawa-san/googolplex-theater/assets/14492530/33cfdc50-55f9-465a-b9cc-5638fddf727d"> - but no luck!

Any help greatly appreciated - thank you.

**Comment by yuzawa-san** [comment]
i think the volume arg needs to take the form  `/path/on/local/volume:/opt/googolplex-theater/conf` but windows uses that `:` so that may throw it off. https://stackoverflow.com/questions/35315996/how-do-i-mount-a-docker-volume-while-using-a-windows-host. so maybe something like this:

```shell
docker run --net=host -v /c/temp:/opt/googolplex-theater/conf yuzawa/googolplex-theater:latest
```

i do not have a windows test environment, so maybe something is broken with the file path discovery within the app itself.

unrelated, but i put off version 2 of this project during the pandemic, but i think maybe given the increased interest i may start testing and preparing for a new release. NOTE: if/when that happens, the config is now a yaml instead of JSON, but no change is needed on your end right now.

**Comment by Trentan** [comment]
Yeah tried that and a few other combos, still no luck - I don't think mounting the file store correctly (where the volume is mapped). Need someone who is a windows expert to get it to work - Looks like i'll invest in a Raspberry Pi. Mind you, I checked and google have allowed streaming from multiple tabs now to separate devices (still can't stream same content to both! need separate tabs!)
